configs/rplidar_process.json

```diff
@@ -0,0 +1,16 @@
+{
+    "remotes": [
+        {
+            "name": "rplidar",
+            "address": "localhost:8081"
+        }
+    ],
+    "processes": [
+        {
+            "id": "rplidar_process",
+            "name": "/home/jeremyhyde-viam/rplidar/main",
+            "cwd": "/home/jeremyhyde-viam/rplidar",
```

Comment:
Personal home directories shouldn't be in config files or code, as it won't work for anyone else (and it's bad practice to "leak" personal data like names.)

Reply:
probably bet to remove config directory and files for this PR

Reply:
Removed. Done.